data/basejob_ui_schema.json

```diff
@@ -0,0 +1,96 @@
+[
+    {
+        "component": "Finput",
+        "model": "jobname",
+        "fieldOptions": {
+        "props": {
+            "label": "Slurm Job Name",
+            "info": "String identifying your job in the SLURM queue"
+        },
+        "on": "input"
```

Comment:
is the `on` here configurable? I wonder if it would be safer to use a distinct name instead of an event type that is also a native event

Reply:
Not that I can tell from [docs](https://jarvelov.gitbook.io/vue-form-json-schema/api/vue-form-json-schema/ui-schema/field/field-options/on)

Reply:
I read that as saying the `on` is configurable and just needs to match your emit.  So you could make it on `formInput` and change your emit to `formInput` and it should _just_work_.

Reply:
or maybe `form-input`? or `forminput`? not sure of the naming convention here